Cargo.toml

```diff
@@ -9,10 +9,12 @@ repository = "https://github.com/palantir/serde-encrypted-value"
 readme = "README.md"
 
 [dependencies]
+aes-gcm = "0.9.4"
```

Comment:
nit: curious why not latest `0.10.1`, delta https://github.com/RustCrypto/AEADs/compare/aes-gcm-v0.9.4...aes-gcm-v0.10.1

Reply:
I wrote the commit a couple of weeks ago before that released - updated!